src/main/scala/beam/agentsim/agents/rideHail/allocationManagers/StanfordRideHailAllocationManagerV1.scala

```diff
@@ -59,7 +61,7 @@ class StanfordRideHailAllocationManagerV1(val rideHailingManager: RideHailingMan
         }
       }
 
-      result = result :+ (vehicleAllocationRequest, vehicleAllocation)
+      result += ((vehicleAllocationRequest, vehicleAllocation))
     }
     result
```

Comment:
return `result.toIndexedSeq` to truly enforce the type

Reply:
If I get this right [TraversableOnce.scala](https://github.com/scala/scala/blob/6ac6da8b61a3a427089a166c7802a940eac71064/src/library/scala/collection/TraversableOnce.scala#L308), toIndexedSeq will allocate new buffer and copy elements there - more GC pressure.